### PR TITLE
fix(loopback4-authorization): CasbinResourceModifierFn definition and readme update

### PR DESCRIPTION
**Pull request by vineet-suri**
SFO-154 SFO-160

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -330,40 +332,51 @@ export class CasbinResValModifierProvider
...
-    const model = path.resolve(
-      __dirname,
-      './../../fixtures/casbin/model.conf',
-    );
+  async action(
+    authUser: IAuthUserWithPermissions,
+    resource: string,
+    isCasbinPolicy?: boolean,
+  ): Promise<CasbinConfig> {
+    const model = path.resolve(__dirname, './../../fixtures/casbin/model.conf');
```
Comment (samarpan-b) [inline README.md:365]: why path from fixtures ?

**Review by samarpan-b — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (vineet-suri) [reply README.md:365]: @samarpan-b Fixtures folder is in the application consuming the extension. We can name it differently also, as you suggest.

Comment (samarpan-b) [reply README.md:365]: hmm. we should ask this data from consumer. path or actual content, whatever. can we do this easily ?

Comment (vineet-suri) [reply README.md]: Done. Updated PR with code to initialise model from file, code or a string. Also updated readme for that.

**Review by samarpan-b — APPROVED** [review]

**Pull request merged**